.changeset/popular-singers-sort.md

```diff
@@ -0,0 +1,21 @@
+---
+'@clerk/clerk-sdk-node': major
+---
+
+Drop deprecations. Migration steps:
+
+- use `CLERK_SECRET_KEY` instead of `CLERK_API_KEY` env variable
+- use `secretKey` instead of `apiKey`
+- use `CLERK_PUBLISHABLE_KEY` instead of `CLERK_FRONTEND_API` env variable
+- use `publishableKey` instead of `frontendApi`
+- drop Redwood hotfix (upgrade to latest version)
+- use `createClerkClient` with options to create a new clerkClient instead of using
+  the following setters:
+    - `setClerkApiVersion`
+    - `setClerkHttpOptions`
+    - `setClerkServerApiUrl`
+    - `setClerkApiKey`
+- use `@clerk/clerk-sdk-node/instance` subpath import instead of `@clerk/clerk-sdk-node/{cjs|esm}/instance`
```

Comment:
To use this the application should have been set with `"moduleResolution": "NodeNext"` (like we did in playground/express application). In order to avoid the change in `moduleResolution` we should use the workaround implemented in `packages/shared` for expo (using the subpaths.mjs` in `postbuild` to generate a folder on the package top level.
cc: @nikosdouvlis

Reply:
We'll already require Node 18 with this new major so requiring `NodeNext` (which resolves to Node 12 or later) is fine, see https://www.typescriptlang.org/tsconfig#moduleResolution

We shouldn't do the subpaths workaround if not absolutely necessary

Reply:
@nikosdouvlis Do you remember the use case for exporting `/instance` as subpath?

Reply:
@dimkl I cover this in my RFC. We're dropping this in v5 so please don't worry about it

Reply:
👍 i will drop it as part of this PR

Reply:
Deprecation added: https://github.com/clerk/javascript/pull/2055